packages/xdg_base.rb

```diff
@@ -3,45 +3,48 @@
 class Xdg_base < Package
   description 'XDG Base Directory Specification Configuration'
   homepage 'https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html'
-  version '0.7-6'
+  version '0.7-7'
   compatibility 'all'
   source_url 'file:///dev/null'
   source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
 
   def self.preinstall
     # Save any previous configuration, if it exists.
-    if File.directory?("#{HOME}/.config") && !File.symlink?("#{HOME}/.config")
-      FileUtils.rm_rf("#{HOME}/.config") unless FileUtils.cp_r("#{HOME}/.config", "#{CREW_PREFIX}/")
+    if File.directory?("#{HOME}/.config") && !File.symlink?("#{HOME}/.config") && !FileUtils.cp_r("#{HOME}/.config",
+                                                                                                  "#{CREW_PREFIX}/")
+      FileUtils.rm_rf("#{HOME}/.config")
     end
-    if File.directory?("#{HOME}/.local") && !File.symlink?("#{HOME}/.local")
-      FileUtils.mkdir_p("#{CREW_PREFIX}/.config") unless Dir.exists? "#{CREW_PREFIX}/.config"
-      FileUtils.rm_rf("#{HOME}/.local") unless FileUtils.cp_r("#{HOME}/.local/.", "#{CREW_PREFIX}/.config/")
+    if File.directory?("#{HOME}/.local") && !File.symlink?("#{HOME}/.local") && !FileUtils.cp_r("#{HOME}/.local/.",
```

Comment:
`"#{HOME}/.local"` without the dot (.)?  We want the directory also, right?  Some package insist on `~/.local` instead of `~/.config`.  It'd be nice if there was some consistency, I know.